.werft/jobs/build/deploy-to-preview-environment.ts

```diff
@@ -153,6 +153,12 @@ export async function deployToPreviewEnvironment(werft: Werft, jobConfig: JobCon
     }
 
     werft.phase(phases.PREDEPLOY, "Checking for existing installations...");
+
+    const maxBranchNameLength = 26;
```

Comment:
A short comment explaining the "magic number" would be nice. Otherwise LGTM! :pray:

Reply:
I could be off, but my arithmetic says that the magic number should be 20.

```
$ echo -n "gitpod.io/registry-facade_ready_ns_staging-" | wc -c
      43
```

Reply:
@jldec Thanks for checking the labels.